samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/CommonMessages_eu.properties

```diff
@@ -55,3 +55,4 @@ cancel_question_info_no_undo=Ekintza hau ezin da berriz desegin!
 cancel_question_info_skip_question=Galdera hau saihestu dezakezu.
 cancel_question_error_cancelling=Errore bat gertatu da galdera ezabatzerakoan. Ezer ez da aldatu.
 cancel_question_info_emi=Kalifikazio doiturik ez dute jasoko "Lotzeko aurreratua" motako elementuek.
+cancel_question_info_random=Questions of assessments with questions that are randomly assigned from question pools can not be cancelled.
```

Comment:
Hi @PabloSanRoman, could you help with this translation, please?

Markus

Reply:
Hello @stetsche , here it is!

cancel_question_info_random= Biltzailetik jasotako galderekin eraikita badago azterketa (ausazko eran), dauden galderak ezin dira baliogabetu.

Reply:
Thank you @PabloSanRoman!